tests/test_pdf2txt.py

```diff
@@ -100,11 +101,31 @@ def test_xml_word(capfd):
 """
 
 
-def test_xml_line(capfd):
+def test_xml_line(capsys):
```

Comment:
If capsys.readouterr() is executed before commandline, the out part will be empty. Thus, it seems like your solution with render temp somehow stores the output of the previous function. Could it be that it is lazy and do not create a new object for each?